index.js

```diff
@@ -226,6 +293,78 @@ Massive.prototype.documentTableSql = function(tableName){
   return sql;
 };
 
+
+Massive.prototype.dropDocumentTable = function(table, options, next) {
```

Comment:
I think calling it `dropTable` instead loses the consistency with `createDocumentTable` but makes it more obvious that it's generally applicable -- there's nothing specific to document tables in the functionality.